modules/cluster-logging-loki-deploy.adoc

```diff
@@ -96,23 +96,23 @@ spec:
   tenants:
     mode: openshift-logging
 ----
-<1> Or `gp2-csi`. 
-+
-.. Apply the configuration:
+<1> Or `gp2-csi`.
+
+. Apply the `LokiStack` CR:
 +
 [source,terminal]
 ----
-oc apply -f logging-loki.yaml
+$ oc apply -f logging-loki.yaml
 ----
-+
-. Create or edit a `ClusterLogging` CR:
+
```

Comment:
It would make sense to add a section to deploy ClusterLogForwarder here since we do mention it under entry-out-of-order-error document and also rate limiting errors are included for audit tenant in it. By default(No CLF) only application and infrastructure logs are sent to Loki.

Reply:
Yeah. To be honest, I think the Operator installs should be separate modules under one assembly (instead of a step in a procedure), and then we need modules on using the APIs from the Operators to create CLF, CL CRs and the same for any Loki CRs. Then those could be pointed to as sort of prerequisite docs.

I didn't want to widen the scope of this PR too much though, because it's supposed to be just for the specific bug fix https://github.com/openshift/openshift-docs/pull/63872/files#diff-da6ea04993d3981274dc8bcaa875a2b2dfc6dc105aa43122676eacd39e10822aR54-R70 and I was doing a bit of general quick cleanup along the way, so let's maybe get this one finished, open an issue for the new section,  and I can come back to that after?